<launch>
	<arg name="arg_w1_x"/>
	<arg name="arg_w1_y"/>
	<arg name="arg_w2_x"/>
	<arg name="arg_w2_y"/>
	<arg name="arg_a0"/>
	<arg name="arg_a1"/>
	<arg name="arg_a2"/>

	<node name="gazebo_info_node" pkg="control_parrot" type="gazebo_info">
	    <param name="param_num" type="int" value="1"/>
	</node>

	<node name="pose_publisher_node" pkg="control_parrot" type="pose_publisher.py" output="screen" launch-prefix="xterm -e">
	    <param name="param_num" type="int" value="1"/>
	    <param name="param_gazebo_info" type="int" value="1"/>
	</node>

	<node name="control_drone_node" pkg="control_parrot" type="control_drone.py" cwd="node" >
	    <param name="param_num" type="int" value="1"/>
		<param name="param_a0" value="$(arg arg_a0)"/>
		<param name="param_a1" value="$(arg arg_a1)"/>
		<param name="param_a2" value="$(arg arg_a2)"/>
	</node>

	<node name="client_node" pkg="control_parrot" type="client.py" output="screen" launch-prefix="xterm -e">
	    <param name="param_num" type="int" value="1"/>
		<param name="param_w1_x" value="$(arg arg_w1_x)"/>
		<param name="param_w1_y" value="$(arg arg_w1_y)"/>
		<param name="param_w2_x" value="$(arg arg_w2_x)"/>
		<param name="param_w2_y" value="$(arg arg_w2_y)"/>
	</node>

</launch>